packages/client/src/pages/customer_area/CustomerSlots/CustomerSlots.tsx

```diff
@@ -110,6 +115,10 @@ const CustomerSlots: React.FC<Props> = ({
   const content = (
     <>
       {countdownProps && <BookingsCountdown {...countdownProps} />}
```

Comment:
This is why I deleted the other BookingsCountdown. It displays a duplicate when there is a second deadline. @ikusteu

Reply:
Yes, that (two notifications onsecond deadline) is the desired behaviour: https://github.com/eisbuk/EisBuk/pull/413